runtime/compiler/env/j9method.cpp

```diff
@@ -8241,6 +8493,20 @@ TR_J9ByteCodeIlGenerator::runFEMacro(TR::SymbolReference *symRef)
          int32_t numNextArguments;
          int32_t spreadStart;
 
+         bool getSpreadPos = symRef->getSymbol()->castToMethodSymbol()->getMandatoryRecognizedMethod() == TR::java_lang_invoke_SpreadHandle_spreadStart ||
```

Comment:
getSpreadPos seems to be used only for JITServer. If that's the case it should be placed inside `#if defined(JITSERVER_SUPPORT)`